### PR TITLE
fix: linting added in hooks [#38]

### PR DESCRIPTION
**Pull request by anjaliahlawat**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,14 @@
   "description": "Starter template for writing JavaScript tests for Gauge",
   "scripts": {
     "test": "./node_modules/.bin/gauge run -p specs/",
-    "lint": "./node_modules/.bin/eslint .",
```
Comment (h2grover) [inline package.json:6]: Please donot remove this line.

File: package.json [hunk 2]
```diff
@@ -3,9 +3,14 @@
...
     "lint:fix": "./node_modules/.bin/eslint . --fix"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "./node_modules/.bin/eslint .",
```
Comment (h2grover) [inline package.json]: Use about command here

**Review by h2grover — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (anjaliahlawat) [reply package.json]: @h2grover I didn't get it. What do you mean by about command?

Comment (h2grover) [reply package.json]: @anjaliahlawat 

Use `yarn run lint` or `lint` and check which one works , instead of `./node_modules/.bin/eslint .`

**Review by h2grover — APPROVED** [review]
LGTM

**Comment by h2grover** [comment]
@anjaliahlawat can you see these errors on your machine?
image

**Comment by anjaliahlawat** [comment]
> @anjaliahlawat can you see these errors on your machine?
> image

In my machine, this command is taking too long to run and giving no output at all. That's why I didn't see these errors. So for the time being, I change the command to run eslint on src/ directory only for making changes in pr. Meanwhile I am trying to find the problem behind this.

**Pull request merged**